frigate/data_processing/common/license_plate/mixin.py

```diff
@@ -1054,13 +1054,20 @@ def lpr_process(self, obj_data: dict[str, any], frame: np.ndarray):
                     for plate in plates
                 )
             ),
-            top_plate,
+            None,
         )
 
-        # Send the result to the API
-        self.sub_label_publisher.publish(
-            EventMetadataTypeEnum.sub_label, (id, sub_label, avg_confidence)
-        )
+        # If it's a known plate, publish to sub_label
```

Comment:
do we want to make this mutually exclusive? Seems to me even if we mark it as a sub label would be fine to keep it with the identifier too

Reply:
Yep, would be good to always have them match. Let me tweak this